File: Class/C1/java/Main.java
class Main
{
    static <_Type> void Reverse(_Type[] list)
    {
        for(int j = 0;j< list.length/2;j++)
                {
                    _Type tmp = list[j];
                    list[j] = list[list.length-j-1];
                    list[list.length-j-1] = tmp;
                }
               
            }
    

    static <_Type> void ReverseOutput(_Type ... list)
    {
        Reverse(list);
                for(_Type i : list)
                {
                    System.out.print(i);
                    if(i != list[list.length-1])
                    System.out.print(',');
                }
        
    }

    // static <_Type> _Type[] GetArray(_Type ... args)
    // {
    //     _Type[] list = new _Type[args.length];
    //     for(int i=0; i<args.length; i++)
    //     {
    //         list[i] = args[i];
    //     }
    //     return list;
    // }

    public static void main(String[] args) {
        ReverseOutput(1,2,3,4,5,6,7,8);
    }
}

File: Class/C1/python/main.py
def reverse(list):
    n = len(list)
    for i in range(0,int(n/2)):
        tmp = list[i]
        list[i] = list[n-i-1]
        list[n-i-1] = tmp

def output_reverse(*args):
    output(args)

def output(list):
    for n in list:
        print(n, end='')
        if n != list[len(list)-1]:
            print(",", end ='')
    print()
if __name__ == "__main__":
    mylist = [1,2,3,4,5]
    reverse(mylist)
    output(mylist)

    output_reverse(1,2,3,4,5,6,7,8,9)